osu.Game/Overlays/Settings/Sections/Graphics/RendererSettings.cs

```diff
@@ -12,6 +14,8 @@ public class RendererSettings : SettingsSubsection
     {
         protected override string Header => "Renderer";
 
+        public override IEnumerable<string> FilterTerms => base.FilterTerms.Concat(new[] { "FPS" });
```

Comment:
not sure i understand

Reply:
This one has been taken from issue suggestion. I'll remove this

Reply:
if it's placed on the FPS display (and wasn't already a keyword) i'd be okay with it.